var/spack/packages/py-pandas/package.py

```diff
@@ -16,6 +16,10 @@ class PyPandas(Package):
     depends_on('py-scipy')
     depends_on('py-setuptools')
     depends_on('py-pytz')
+    depends_on('libdrm')
+    depends_on('libpciaccess')
+    depends_on('llvm')
```

Comment:
I believe @karenyyng was getting build failures complaining about the lack of llvm and added it.